mergify/src/main/scala/org/typelevel/sbt/mergify/MergifyAction.scala

```diff
@@ -59,16 +61,30 @@ object MergifyAction {
       }
   }
 
-  final case class RequestReviews(users: List[String] = Nil) extends MergifyAction {
+  final class RequestReviews(val users: Either[Seq[String], Map[String, Int]], val randomCount: Option[Int]) extends MergifyAction {
```

Comment:
> Also I can't remember how Either renders by default with Circe

I folded them together so that if it's a list it gets rendered without the weights—I was thinking it would be kind of confusing to have weights if they're not specified. I know it's not something users should be editing, but it's still visible and people could be spot-checking it or copy/pasting it into the Mergify config editor for validation or to see what rules would be triggered on different PRs.

Reply:
Ok, fair point. Let's keep it then. How does Circe generate yaml from `JSON`?

Actually, you can add a `RequestReviews` action to the sbt-typelevel mergify config in `build.sbt` so we can dog-food this :)